hosts

```diff
@@ -237,6 +237,7 @@ qld-nvme ansible_ssh_host=203.101.231.166
 
 [stats_servers]
 stats.usegalaxy.org.au ansible_ssh_host=115.146.84.128
+stats-2025.usegalaxy.org.au ansible_ssh_host=115.146.87.139
```

Comment:
Can this be called stats-2025 instead? the old stats has an awkward hostname. I see that you have tried not to double up on playbooks but maybe we need to until the existing VM is no longer needed.

Reply:
Yep. will do!

Reply:
The stats playbook also has patterns that we don’t normally use (which are valid patterns) of implicit variables files. Listing every file is a style we agreed on a while ago but it wasn’t done for this one - it means a person does not need to have memorised the structure of the hosts file to know where to find the variables.

Reply:
Yep. I'll update the new playbook.